app/helpers/curation_concerns/file_set_helper.rb

```diff
@@ -7,9 +7,9 @@ def parent_path(parent)
     end
   end
 
-  def media_display(file_set, locals = {})
-    render media_display_partial(file_set),
-           locals.merge(file_set: file_set)
+  def media_display(presenter, locals = {})
```

Comment:
Should this method name be `#file_set_media_display`? If so, for now, an alias could be added and deprecate the existing media_display.

The reason I ask is are there other `media_display` helper methods? If there are, these could be mixed into the same view and obliterate each other.